deepcave/runs/converters/amltk.py

```diff
@@ -93,11 +95,9 @@ def from_path(cls, path: Union[Path, str]) -> "AMLTKRun":
         path = Path(path)
 
         # Read configspace
-        from ConfigSpace.read_and_write import json as cs_json
-
         with open(path / "configspace.json", "rb") as f:
             json_string = pickle.load(f)
-        configspace = cs_json.read(json_string)
+        configspace = ConfigurationSpace.from_json(StringIO(json_string))
```

Comment:
Interesting that you read it as bytes with `"rb"`.

You could however just do `ConfigurationSpace.from_json(path / "configspace.json")` if your file is actually json (and not byte encoded)

Reply:
Oh I see you've done that elsewhere, interesting that it's specifically here

Reply:
Thanks for checking!

Yes, the reason for this was that when saving the configspace in AMLTK via
`bucket.store({"configspace.json": cs_json.write(space)})`
I got back an encoded json file. Not sure if that is the correct way to save the json there then though?

Reply:
Oh, should have just been `bucket.store({"configspace.json": space.to_serialized_dict()})`, [ref](https://github.com/automl/ConfigSpace/blob/810c8d8a2955f36b91e89a8f2a924a0f74bc6a67/src/ConfigSpace/configuration_space.py#L952) then!

The `cs_json.write(space)` converts it to a binary string, hence the issue